politeiawww/cmsdatabase/database.go

```diff
@@ -61,6 +61,11 @@ type Invoice struct {
 	UserSignature      string
 	ServerSignature    string
 	Version            string // Version number of this invoice
```

Comment:
Move version to be the first item of struct to remain consistent with rest of code.